macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/internal/DefaultPlantUMLConfiguration.java

```diff
@@ -56,4 +56,10 @@ public String getPlantUMLServerURL()
         }
         return serverURL;
     }
+
+    @Override
+    public String getPlantUMLOutputFormat()
+    {
+        return this.xwikiPropertiesConfigurationSource.getProperty("plantuml.format");
```

Comment:
You can use the `fromString()` here to return a `PlantUMLDiagramFormat` object.

Reply:
BTW you could also call: `this.xwikiPropertiesConfigurationSource.getProperty("plantuml.format", "png");` which is also a bit nicer (it says to default to "png" if "plantuml.format" doesn't exist.